package.json

```diff
@@ -122,5 +123,11 @@
   "repository": {
     "type": "git",
     "url": "git@github.com:liquality/wallet.git"
+  },
+  "resolutions": {
```

Comment:
@bradleySuira hahaha... Just a small not so we can remember to remove this before merging.